### PR TITLE
Release 4.3.0

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.3.0 92020-09-15)
+
+- Added a message stream to interact with the back to top directive programatically and options to not display the back to top button. [#158](https://github.com/blackbaud/skyux-layout/pull/158)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Added a message stream to interact with the back to top directive programatically and an option to hide the back to top button. [#158](https://github.com/blackbaud/skyux-layout/pull/158)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 4.3.0 92020-09-15)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:1]:
```suggestion
# 4.3.0 92020-09-16)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# 4.3.0 92020-09-16)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:1]:
```suggestion
# 4.3.0 (2020-09-17)
```

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.3.0)

**Pull request merged**